js_modules/dagster-ui/packages/ui-components/src/components/Toaster.tsx

```diff
@@ -71,6 +71,15 @@ const setup = (instance: ToasterInstance): DToaster => {
         </>
       );
     }
+    if (rest?.action?.href) {
```

Comment:
This is an interesting fix but I am worried this behavior is inconsistent with the rest of the UI kit -- if you make an `<a href=` or a `window.open(href)` call, you have to pass the correct href. Why should toaster let you pass an incorrect href and fix it for you?

We have some places like `RunUtils.tsx` where we already pass the href correctly built from `history.createHref`. Can you update the callsites that are doing it incorrectly to do it right instead?

I'm worried that these places that ARE doing it right will have the href "double prepended" with the path prefix now.

<img width="713" alt="image" src="https://github.com/dagster-io/dagster/assets/1037212/7743ad8d-94c9-4039-8ce7-89fb63f75f2f">

Reply:
Alternatively, we could extend the Toaster data to let you specify `action: {text: string, to: string}` and then have the toaster transform the "to" into an href. I think that'd be consistent with how `<Link` works and would be nice!

Reply:
> We have some places like RunUtils.tsx where we already pass the href correctly built from history.createHref. Can you update the callsites that are doing it incorrectly to do it right instead?

Yeah I was considering doing that too but the utilities creating the href aren't hooks so they can't access `createHref`.  I guess I can update their callsites to pass the history object from `useHistory` to them. We could also check if it already has the prefix and if so skip adding it.